include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -414,6 +421,17 @@ class BlmcJointModules
         }
     }
 
+
+    Vector get_max_joint_torques()
```

Comment:
naming is incoherent with function above for getting a single max_torque, i.e. it should be get_max_torques()

Reply:
I agree with you I just tried to change to max_torque but then it conflicts with the RealDisentanglementPlatform::get_max_torques()

Reply:
ok, if it is just related to DisentanglementPlatform (and not Finger) then feel free to just disable that code, it's outdated anyway. actually, @luator, maybe we could just remove all the disentanglement related stuff for now.

Reply:
What is `get_max_torques` doing? If it's doing the same, can it be overloaded in this class?

Reply:
> What is `get_max_torques` doing? If it's doing the same, can it be overloaded in this class?

I am not sure I understand your comment @jviereck ... get_max_torqueS return all max_torqueS form the joint. get_max_torque return 1 maximum torque as a double.

Reply:
@mwuethri should I keep the files and just remove them from the CMakeLists.txt? This way I leave to you the deletion and cleaning of the code?

Reply:
Oo just removing the demo_disentanglement_platform.cpp from the compilation worked.

Reply:
hmm actually this might break some of our code. could you maybe just create an additional function which is called get_max_torques() and we leave the get_max_joint_torques() as a deprecated function for now?

then maybe you can create an issue and assign it to felix and me to get rid of the get_max_joint_torques() function.

Reply:
Solved by recent commit.